api/api.go

```diff
@@ -270,6 +270,10 @@ func (c *Client) Status(trackID string) (*StatusInfo, error) {
 
 	defer res.Body.Close()
 
+	if _, err := c.List(trackID); err != nil {
+		return nil, err
+	}
+
```

Comment:
OK, I think that the fix is to improve that endpoint so that it says whether or not the track exists. Are you comfortable in Ruby? If not, no worries, I can do this easily.